packages/blockchain-wallet-v4-frontend/src/data/components/request/sagas.ts

```diff
@@ -30,7 +32,11 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
       let address
       let extras: RequestExtrasType = {}
       const { account } = action.payload
-
+      const subscriptions = yield select(S.getSubscriptions)
```

Comment:
`const subscriptions = S.getSubscriptions(yield select())`

this will maintain type-safety